zone/client_packet.cpp

```diff
@@ -5280,28 +5280,55 @@ void Client::Handle_OP_ConsiderCorpse(const EQApplicationPacket *app)
 		}
 	}
 
-	uint32 decay_time = t->GetDecayTime();
-	if (decay_time) {
-		const std::string& time_string = Strings::SecondsToTime(decay_time, true);
-		Message(
-			Chat::NPCQuestSay,
-			fmt::format(
-				"This corpse will decay in {}.",
-				Strings::ToLower(time_string)
-			).c_str()
-		);
+	if (t && t->IsNPCCorpse()) {
+		uint32 min; uint32 sec; uint32 ttime;
+		if ((ttime = t->GetDecayTime()) != 0) {
+			sec = (ttime / 1000) % 60; // Total seconds
+			min = (ttime / 60000) % 60; // Total seconds / 60 drop .00
```

Comment:
Let's clean this up a bit, `sec` and `min` are not helping readability. If we slightly expand the names we no longer need the comments `// Total seconds` because it describes itself.

```cpp
		uint32 minute;
		uint32 second;
		uint32 decay_time = t->GetDecayTime();
		if (decay_time != 0) {
```